src/API/VendorAPI.php

```diff
@@ -54,6 +54,20 @@ class VendorAPI {
 	 */
 	public $endpoint_callback;
 
+	/**
+	 * Specify if the endpoint supports multiple methods
+	 *
+	 * @var bool
+	 */
+	public $supports_multiple_endpoints = false;
+
+	/**
+	 * Map with callbacks for each supported method
+	 *
+	 * @var array
+	 */
+	public $endpoint_callbacks_map = array();
```

Comment:
This should be `protected`, as we don't need `public` access.

```suggestion
	protected $endpoint_callbacks_map = array();
```